coalib/results/result_actions/OpenEditorAction.py

```diff
@@ -34,7 +34,7 @@ def is_applicable(result, original_file_dict, file_diff_dict):
 
     def apply(self, result, original_file_dict, file_diff_dict, editor: str):
         '''
-        Open the affected file(s) in an editor.
+        Open in editor.
```

Comment:
shave every word, in this case `in`, I trust you can shave on the others as well ;)